core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -1735,6 +1735,13 @@ public void removeNode(@Nonnull Node n) throws IOException {
         nodes.removeNode(n);
     }
 
+    /**
+     * Updates an existing {@link Node} on disk.
+     */
+    public void updateNode(Node n) throws IOException {
```

Comment:
return a boolean esp. given the semantics of the update operation if there is an existing node of the same name that is not the provided node instance